libsolidity/lsp/Transport.cpp

```diff
@@ -107,8 +107,15 @@ void IOStreamTransport::send(Json::Value _json, MessageID _id)
 
 	string const jsonString = solidity::util::jsonCompactPrint(_json);
 
-	m_output << "Content-Length: " << jsonString.size() << "\r\n";
-	m_output << "\r\n";
+#if defined(_WIN32)
```

Comment:
I don't think this is a proper solution. I saw some posts mentioning freopen and others `setmode`: https://docs.microsoft.com/en-us/cpp/c-runtime-library/reference/setmode?view=msvc-170&viewFallbackFrom=vs-2019